distrobox-create

```diff
@@ -308,10 +318,14 @@ fi
 # First, check if the image exists in the host.
 # If not prompt to download it.
 if ! ${container_manager} inspect --type image "${container_image}" >/dev/null 2>&1; then
-	# Prompt to download it.
-	printf >&2 "Image not found.\n"
-	printf >&2 "Do you want to pull the image now? [y/n] "
-	read -r response
+	if [ ${non_interactive} -eq 0 ]; then
```

Comment:
We should quote this, linting will fail with:

`Prefer double quoting even when variables don't contain special characters. [SC2248]`